src/components/menu-bar/menu-bar.css

```diff
@@ -197,3 +200,22 @@
     filter: hue-rotate(360deg);
   }
 }
+
+.mystuff > a {
```

Comment:
I don't think the `a` tag describes this sufficiently and could use a class name. (Can happen in a follow up)

Reply:
Made https://github.com/LLK/scratch-gui/issues/3213